installation.md

```diff
@@ -152,11 +152,55 @@ with `M-x company-mode`.
 - **Debug logs**: available in the `EGLOT events` buffer.
 - **Command-line flags and alternate binary**: instead of adding `"clangd"`
   to `eglot-server-programs`, add `("/path/to/clangd" "-log=verbose")` etc.
+
+---
+
+[lsp-mode](https://github.com/emacs-lsp/lsp-mode) can also be configured to work with clangd. It has support for all standard clangd features and some of the extensions, like completion scores, or type hierarchies.
+
+Install lsp-mode with `M-x package-install RET lsp-mode RET`.
+
+Add the following to `~/.emacs` to enable lsp-mode:
+
+```
+(require 'lsp-mode)
+(add-hook 'c-mode-hook #'lsp)
+(add-hook 'c++-mode-hook #'lsp)
+```
+
+### Indexing status
+
+lsp-mode can process indexing notifications from clangd and show you progress in the mode line.
+
+![Indexing status in lsp-mode](screenshots/lsp_mode_indexing.png)
+
+### Clang-tidy error explanations
+
+The clangd Flycheck integration in lsp-mode shows error explanations from the LLVM website for clang-tidy linting issues.
+This is very useful when you need context before applying a clang-tidy fix. Press `M-x flycheck-explain-error-at-point` when the point is at a clang-tidy error.
+
+![clang-tidy error explanations in lsp-mode](screenshots/lsp_mode_clang_tidy_explanations.png)
+
+### Type signatures in mode line
+
+When the point is at a symbol, lsp-mode extracts a function prototype and shows it in the mode line. Simply placing the
+point on top of an auto variable, for example, will show its type.
+
+![Mode line signatures in lsp-mode](screenshots/lsp_mode_signature.png)
+
+### Find workspace symbols
+
+You can search for workspace symbols using a package like [lsp-ivy](https://github.com/emacs-lsp/lsp-ivy) or [helm-lsp](https://github.com/emacs-lsp/helm-lsp)
+
+![Find workspace symbols in lsp-mode](screenshots/lsp_mode_workspace_symbols.png)
+
+### Additional UI features
+
+Additional UI features like peek references, peek definition, etc. are available as a separate package, [lsp-ui](https://github.com/emacs-lsp/lsp-ui)
 </details>
```

Comment:
we need to have the "under the hood" section briefly explaining how to get to clangd logs and how to set clangd flags/binary path. Other parts of the docs (and the bug tracker) assume the user knows how to do this in their editor.

Reply:
Done.